pyClient/test/test_contract_base_mixer.py

```diff
@@ -60,20 +60,16 @@
 # h_sig = "111" = 7
 # nf_0  = "000" = 0
 # nf_1  = "001" = 1
-# cm_0  = "010" = 2
-# cm_1  = "011" = 3
-# h_0   = "100" = 4
-# h_1   = "101" = 5
+# h_0   = "100" = 2
```

Comment:
This is 4. Should be `h_0   = "010" = 2`

Reply:
Updated.